pkg/fakerp/fakerp.go

```diff
@@ -91,8 +91,23 @@ func createOrUpdate(ctx context.Context, log *logrus.Entry, cs, oldCs *api.OpenS
 		return nil, kerrors.NewAggregate(errs)
 	}
 
+	template, err := GetPluginTemplate()
+	if err != nil {
+		return nil, err
+	}
+
+	// if running under test we can overwrite certain images from local env variables
+	if config.TestConfig.RunningUnderTest {
+		overridePluginTemplate(template)
+	}
+
+	errs = p.ValidatePluginTemplate(ctx, template)
```

Comment:
can you move this up to next to plugin.NewPlugin above?
can it be done once when the fakerp is initialised, not per createUpdate?

Reply:
Will move this up. At the moment `createOrUpdate` represents RP process. we migh want to change name to make it more representative